newScript.js

```diff
@@ -0,0 +1,63 @@
+var divs = new Array();
+    divs[0] = "errFirst";
```

Comment:
Please append a `var` to each of these variable declarations.  Better yet, please use `.push()` instead.